File: CalculoCDB.Domain/Request/CalculoCdbRequest.cs
using System.ComponentModel.DataAnnotations;

namespace CalculoCDB.Domain.Request
{
    public class CalculoCdbRequest
    {
        [Required]
        [DataType(DataType.Currency)]
        [Range(1, int.MaxValue)]
        public decimal ValorInvestimento { get; set; }

        [Required]
        [Range(1, int.MaxValue)]
        public int PrazoMeses { get; set; }
    }
}

File: Site/CalculoCDB/src/model/CalculoCdbRequest.ts
export class CalculoCdbRequest {
    ValorInvestimento:number
    PrazoMeses:number

    constructor(ValorInvestimento: number, PrazoMeses: number) {
        this.ValorInvestimento = ValorInvestimento;
        this.PrazoMeses = PrazoMeses;
      }
  }